### PR TITLE
project datatable refresh issue

### PR DESCRIPTION
**Pull request by Shannibrst**

### CODE REVIEW

File: imports/startup/client/routes/projects.js [hunk 1]
```diff
@@ -40,11 +40,6 @@ Router.route(
         return Roles.userIsInRole(Meteor.userId(), DefaultAdminAccessRoles);
       },
     },
-    waitOn() {
-      return [
-        Meteor.subscribe('projects.all'),
```
Comment (art4ride) [inline imports/startup/client/routes/projects.js]: Why did you remove projects subscription?

Comment (Shannibrst) [reply imports/startup/client/routes/projects.js]: There is no need for subscription on create project page and after removing subscription everything looks working fine.
